code/src/main/java/com/google/sps/data/Business.java

```diff
@@ -0,0 +1,31 @@
+package com.google.sps.data;
+
+import com.googlecode.objectify.annotation.Entity;
+import com.googlecode.objectify.annotation.Id;
+import java.time.Instant;
+
+@Entity
+public class Business {// This class is solely to check if a business was contacted in the last week.
+
+  @Id String placeId;
+  private long timeOfLastContactUnixTimeSeconds;
```

Comment:
UnixTime here is redundant, since time is in the name, and seconds is at the end. Assuming the Unix Epoch is reasonable since all computer systems use it. 

timeOfLastContactSeconds is ideal here.

Reply:
Okay I changed everything form UnixTimeSeconds to Seconds.

Reply:
Thanks!